pennylane/ops/op_math/controlled_class.py

```diff
@@ -177,7 +180,7 @@ def __init__(
                     "Specifying control values as a string is deprecated. Please use Sequence[Bool]",
                     UserWarning,
                 )
-                control_values = [(x == "1") for x in control_values]
+                control_values = [(x != "0") for x in control_values]
```

Comment:
Why this change?

Reply:
See Albert's comment above (just marked as unresolved, so should be visible now)

Reply:
Ah, so everything that's not `"0"` gets cast to `True`. Might be good to add a comment as to that behaviour, as this could be a potential source of confusion.